test/testing_result.py

```diff
@@ -0,0 +1,76 @@
+"""
+Testing Mandyoc code using Crameri example for only 3 time steps
+"""
+
+import numpy as np
+import os
+
+# Test path
+test_path = "test/testing_result_data"
+# Path to the expected data to make the comparison
+expected_result_path = os.path.join(test_path, "expected_result")
+# Name of the files to compare
+file_name = [
+    "density_{}", 
+    "heat_{}",
+    "pressure_{}",
+    "strain_{}",
+    "strain_rate_{}",
+    "temperature_{}",
+    #"sp_surface_global_{}",
+    "viscosity_{}",
+    "step_{}_0",
+    "step_{}_1",
+    "time_{}",
+    "velocity_{}",
+]
+# steps to compare
+steps = [0, 10, 20]
```

Comment:
I'm not sure if we really need to compare various time steps here. If results goes out bounds expected values in the begging then I don't see it converging for later steps.
So, I suggest we only compare only one later step, cause if it not pass the test then it was wrong from the beginning.

Also, this would help decreasing testing time. I'm suggesting 0 and 1 here, but we can play around to see how it goes. Of course that require some work on files in expected result folder and `param.txt` file.

```suggestion
steps = [0, 1]
```